src/SFA.DAS.EmployerApprenticeshipsService.Web.UnitTests/Orchestrators/EmployerCommitmentOrchestrator/WhenCreatingEmployerAssignedCommitment.cs

```diff
@@ -80,5 +88,48 @@ public void Arrange()
                 Times.Once);
         }
 
+
+        [Test]
+        public async Task ShouldCallTransferMappingFunction()
+        {
+            //Act
+            await EmployerCommitmentOrchestrator.CreateEmployerAssignedCommitment(_viewModel, "externalUser",
+                "DisplayName", "UserEmail");
+
+            //Assert
+            MockCommitmentMapper.Verify(x => x.MapToTransferConnectionsViewModel(_getTransferConnectionsResponse.TransferConnections));
+        }
+
+        [Test]
+        public async Task ShouldMapPropertiesToCommandWithoutTransferConnection()
+        {
+            //Act
+            _viewModel.TransferConnectionCode = null;
+            await EmployerCommitmentOrchestrator.CreateEmployerAssignedCommitment(_viewModel, "externalUser",
+                "DisplayName", "UserEmail");
+
+            //Assert
+            MockMediator.Verify(x => x.SendAsync(
+                    It.Is<CreateCommitmentCommand>(p => p.UserId == "externalUser" &&
+                                                        p.Commitment.Reference == _viewModel.CohortRef &&
```

Comment:
Using _viewModel's properties as the 'expected' values means the sut could alter the _viewModel's properties and the test success could be a false positive.

If you clone _viewModel before calling the sut, you could pick the expected values from the clone (which the sut has no opportunity to mutate).

Reply:
This was just checking a property. Very, very low risk